### PR TITLE
Display 403 failures on action/flow delete

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] Add the Clubhouse Story ID: [ch23105]

### CODE REVIEW

**Comment by LisaManresa** [comment]
This is pending what we decide to do with worklist bulk delete. But we can get 👀  on everything else

File: src/js/apps/patients/worklist/worklist_app.js [hunk 1]
```diff
@@ -255,11 +255,18 @@ export default App.extend({
       },
       'delete'() {
         const itemCount = this.selected.length;
-        this.selected.destroy().then(() => {
-          this.showDeleteSuccess(itemCount);
-          app.stop();
-          this.getState().clearSelected();
-        });
+
+        this.selected.destroy({ wait: true })
```
Comment (paulfalgout) [inline src/js/apps/patients/worklist/worklist_app.js]: I think we should revert this, so it's clear _something_ needs to be done.. I don't think this'd what it'd be if we do this.

File: test/integration/patients/sidebar/flow-sidebar.js [hunk 2]
```diff
@@ -373,8 +407,6 @@ context('flow sidebar', function() {
       .find('.js-submit')
       .click();
 
-    cy
```
Comment (paulfalgout) [inline test/integration/patients/sidebar/flow-sidebar.js:376]: should this get removed?

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: test/integration/patients/sidebar/flow-sidebar.js (hunk 2 above)
Comment (LisaManresa) [reply test/integration/patients/sidebar/flow-sidebar.js:376]: No it should not.

**Review by paulfalgout — APPROVED** [review]

**Pull request merged**